lib/chewy/index/actions.rb

```diff
@@ -225,6 +225,10 @@ def clear_cache(args = {index: index_name})
           client.indices.clear_cache(args)
         end
 
+        def _reindex(source_index, dest_index)
+          client.perform_request('POST', '_reindex', {}, {source: {index: source_index}, dest: {index: dest_index}})
+        end
```

Comment:
I would call this method `reindex` without underscore and also would take key params like this:

```suggestion
        def reindex(source: index_name, destination: index_name)
          client.perform_request(
            'POST',
            '_reindex',
            {},
            {
              source: {index: source},
              dest: {index: destination}
            }
          )
        end
```

Also having `index_name` as default for both, helps people to set only one of them when is desired to apply from or to default index name (that I believe that might be the most common use case)

Reply:
Also it would be nicer to use [reindex](https://www.rubydoc.info/gems/elasticsearch-api/Elasticsearch/API/Actions#reindex-instance_method) method provided by ES Ruby gem